scripts/verify-brackets.js

```diff
@@ -0,0 +1,14 @@
+const config = require("../server/routes/config");
+const isReachable = require("is-reachable");
+const brambleServer = config.editorHOST;
+const colors = require("colors");
+
+isReachable(brambleServer).then(reachable => {
+  if (!reachable) {
+    console.warn(
+      colors.yellow(
+        `Error: Bramble server is not running. Please end this process and run the Brackets server with: npm start`
```

Comment:
ah darn, sorry I missed this, could you change this to:
```
`Error: Brackets is not running. In a separate terminal window, run Brackets using npm start in the brackets folder.`
```

Reply:
No worries, I'm on it 👍